src/transport_ofi.c

```diff
@@ -1133,6 +1359,47 @@ int shmem_transport_init(void)
     ret = allocate_fabric_resources(&shmem_transport_ofi_info);
     if (ret != 0) return ret;
 
+    /* STX max settings */
+    if ((shmem_internal_thread_level == SHMEM_THREAD_SINGLE ||
+         shmem_internal_thread_level == SHMEM_THREAD_FUNNELED ) &&
+         shmem_internal_params.OFI_STX_MAX > 1) {
+        if (shmem_internal_params.OFI_STX_MAX_provided) {
+            /* We need only 1 STX per PE with SHMEM_THREAD_SINGLE or SHMEM_THREAD_FUNNELED */
+            RAISE_WARN_MSG("Ignoring invalid STX max setting '%ld' w/THREAD_SINGLE, using 1\n",
```

Comment:
This applies to both single and funneled.  Suggest updating the message to: "Ignoring STX max setting '%ld'; using 1 STX in single-threaded mode".